src/base/utils/net.h

```diff
@@ -28,13 +28,23 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
 class QString;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+
+        template <typename T>
+        T unquote(const T &str)
```

Comment:
@evsh, @Chocobo1, you had a long discussion about it, but I did not understand the following (maybe I missed something):
1. Why this function is in inappropriate place? (or why it has the wrong name?) Function name (full name) should speak for itself. `Utils::String::unquote` fully reflects its meaning (unquote string). But what does `Utils::Net::unquote` mean? Unquote WHAT?
2. Why this function has unexpected behavior? Let's start with the concept of quoted string. What is it? Quoted string is 0 or more characters enclosed in any kind of quotes. I.e. it must be of length at least 2 characters (`"some text"` is quoted string, `""` is also quoted *empty* string, but `"` isn't).